clients/benchmarks/client.cpp

```diff
@@ -290,6 +296,7 @@ struct perf_blas<T,
                 {"iamax", testing_iamax<T>},
                 {"iamin", testing_iamin<T>},
                 {"gemv", testing_gemv<T>},
+                {"trmv", testing_trmv<T>},
```

Comment:
Complex batched/strided_batched?

Reply:
gemv batched / strided batched?

Reply:
Sure, those too. Just should add them so we can do benchmark tests for complex values for all functions that support it.

Reply:
Added